frontend/src/metabase/lib/query.js

```diff
@@ -237,222 +229,6 @@ export function getFieldOptions(
   return results;
 }
 
-export function formatField(fieldDef, options = {}) {
-  const name = stripId(fieldDef && (fieldDef.display_name || fieldDef.name));
-  return name;
-}
-
-export function getFieldName(tableMetadata, field, options) {
-  try {
-    const target = FIELD_REF.getFieldTarget(field, tableMetadata);
-    const components = [];
-    if (target.path) {
-      for (const fieldDef of target.path) {
-        components.push(formatField(fieldDef, options), " → ");
-      }
-    }
-    components.push(formatField(target.field, options));
-    if (target.unit) {
-      components.push(` (${target.unit})`);
-    }
-    return components;
-  } catch (e) {
-    console.warn(
-      "Couldn't format field name for field",
-      field,
-      "in table",
-      tableMetadata,
-    );
-  }
-  return "[Unknown Field]";
-}
-
-export function getTableDescription(tableMetadata) {
-  return [inflection.pluralize(tableMetadata.display_name)];
-}
-
-export function getAggregationDescription(tableMetadata, query, options) {
```

Comment:
I think this will conflict with Rob's changes that are in #12665. I think your changes for this file are just moving things around, so it shouldn't be too hard to sort out.